controllers/actions.github.com/ephemeralrunner_controller.go

```diff
@@ -43,7 +43,8 @@ const (
 	// It represents the name of the container running the self-hosted runner image.
 	EphemeralRunnerContainerName = "runner"
 
-	ephemeralRunnerFinalizerName = "ephemeralrunner.actions.github.com/finalizer"
+	ephemeralRunnerFinalizerName        = "ephemeralrunner.actions.github.com/finalizer"
+	ephemeralRunnerActionsFinalizerName = "ephemeralrunner.actions.github.com/actions"
```

Comment:
`ephemeralrunner.actions.github.com/runnerRegistrationFinalizer` ?

Reply:
How do you feel about: `ephemeralrunner.actions.github.com/runner-registration-finalizer`